rotkehlchen/tests/db/test_history_events.py

```diff
@@ -219,9 +219,9 @@ def test_read_write_events_from_db(database):
 
 
 @pytest.mark.parametrize(
-    ('has_premium', 'group_by_event_ids'), [(True, False), (True, True), (False, False), (False, True)],  # noqa: E501
+    ('entries_limit', 'group_by_event_ids'), [(None, False), (None, True), (FREE_HISTORY_EVENTS_LIMIT, False), (FREE_HISTORY_EVENTS_LIMIT, True)],  # noqa: E501
 )
-def test_read_write_customized_events_from_db(database: DBHandler, has_premium: bool, group_by_event_ids: bool) -> None:  # noqa: E501
+def test_read_write_customized_events_from_db(database: DBHandler, entries_limit: bool, group_by_event_ids: bool) -> None:  # noqa: E501
```

Comment:
why did this not hit mypy?

```suggestion
def test_read_write_customized_events_from_db(database: DBHandler, entries_limit: int, group_by_event_ids: bool) -> None:  # noqa: E501
```

Reply:
the test is parametrised so mypy doesn't know the real type.